x-pack/plugins/security_solution/public/timelines/components/side_panel/event_details/flyout/back_to_alert_details_link.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiButtonEmpty, EuiText, EuiTitle } from '@elastic/eui';
+import React from 'react';
+import {
+  ISOLATE_HOST,
+  UNISOLATE_HOST,
+} from '../../../../../detections/components/host_isolation/translations';
+import { ALERT_DETAILS } from '../translations';
+
+const BackToAlertDetailsLinkComponent = ({
```

Comment:
should we delete the existing versions of this and the other flyout components in this pr as well?

Reply:
nvm I see you basically did below